campaignion_manage/src/ContentPage.php

```diff
@@ -47,6 +48,9 @@ public function __construct(Content $query) {
       $ops['publish'] = new ContentPublish();
       $ops['unpublish'] = new ContentUnpublish();
     }
+    if (module_exists('campaignion_manage_clone')) {
```

Comment:
Wouldn’t `use Drupal\campaignion_manage_clone\ContentClone;` throw an error if the module didn’t exist?

Reply:
Ok, fine then :) I also managed to find a comment on the documentation page explaining this (there is nothing in the documentation): https://www.php.net/manual/en/language.namespaces.importing.php#121045